samples/Mvc.LocalizationSample.Web/SelfContainedValidationAttribute.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Mvc.DataAnnotations;
+using Microsoft.AspNetCore.Mvc.ModelBinding.Validation;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Localization;
+using Microsoft.Extensions.Options;
+
+namespace Mvc.LocalizationSample.Web
+{
+    public abstract class SelfContainedValidationAttribute : ValidationAttribute, IClientModelValidator
+    {
+        public abstract void AddValidation(ClientModelValidationContext context);
```

Comment:
Because otherwise `SelfContainedValidationAttribute` wouldn't completly implement `IClientModelValidator`. Do you have another method of doing this that you recomend?